packages/plugin-lighthouse/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@quality-metrics/lighthouse-plugin",
+  "version": "0.0.1",
+  "dependencies": {
+    "lighthouse": "^11.0.0"
```

Comment:
Should this not happen automatically by nx

Reply:
Addressed above.